.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Types of changes
+<!--
+  What types of changes does your code introduce? Put an `x` in all the boxes
+  that apply:
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality
+      to change)
+
+## Checklist:
+<!--
+  Go over all the following points, and put an `x` in all the boxes that apply.
+-->
+- [ ] I run eslint on the code
+- [ ] I have added JSDoc for all of my code (where applicable)
+- [ ] I have added tests to cover my changes.
+
+## Priority:
+- [ ] Normal <!-- New piece of functionality -->
```

Comment:
I wondered about that, and actually put it in at first, but changed my mind since PRs are supposed to be high priority generally (for review)...if you see what I mean

Reply:
That's a fair point. @chrxr should probably weigh in on this, but think there are cases where we may have a low-priority change (e.g. tweaking minor config files). I guess I think about it in terms of "how soon does this need to be reviewed?", along the lines of:

**high** drop everything and review this now
**normal** finish what you're doing and review this as soon as you can
**low** review this sometime this week